src/pages/codingClubs/CodingClubs.tsx

```diff
@@ -0,0 +1,99 @@
+import { type FC } from "react"
+import { Typography, useTheme } from "@mui/material"
+import { Download as DownloadIcon } from "@mui/icons-material"
+
+import * as page from "codeforlife/components/page"
+
+import CodeClubHeroImage from "../../images/coding_club_hero_hexagon.png"
+import AboutUsImage from "../../images/about_us.jpg"
+import PythonClubImage from "../../images/coding_club_python_pack.png"
+import Introduction from "../../components/Introduction"
+
+import ClubAim from "./ClubAim"
+import { Link, LinkButton } from "codeforlife/components/router"
```

Comment:
Sort imports. Unfortunately, prettiers import sorting works differently to isort in python (prettier's behaviour is worse IMO). Prettier sorts imports by group (not as a whole). The sort order I have been following so far is:

```tsx
// {default and 3rd party imports on top}
// space here
// {codeforlife imports in the middle}
// space here
// {relative imports on the bottom}
```

If we want to change this order, we need to so everywhere.